File: e2e/selenide-junit5-selenium-grid/src/test/java/test/mobile/deleteTask/DeleteTaskTest.java
package test.mobile.deleteTask;

import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import io.qameta.allure.TmsLink;
import test.base.MobileTest;
import test.spec.deleteTask.IDeleteTaskTest;

import static data.AllureEpic.TASK_MANAGEMENT;

@Epic(TASK_MANAGEMENT)
@Feature("Delete task")
@TmsLink("99")
class DeleteTaskTest extends MobileTest implements IDeleteTaskTest {}

File: e2e/maestro/tests/delete-task/delete-task.test.ts
import { AllureEpic } from '@/data/AllureEpic';
import { maestroTestCase } from '@/runner/maestroTestCase';

export default maestroTestCase(import.meta.url, {
  name: 'should remove task from list when delete succeeds',
  epic: AllureEpic.TASK_MANAGEMENT,
  feature: 'Delete task',
  tms: '99',
  suite: 'mocked',
  setupMocks: async (support, context) => {
    const response = support.mock.api.fromResponse(context.createTaskResponse());

    await support.reset();
    await support.mock.api.getTasks(response);
    await support.mock.api.getTasks(response);
    await support.mock.api.deleteTask();

    return context.toMaestroEnv();
  },
});
